entries/p/psnprofiles.com.json

```diff
@@ -1,11 +1,10 @@
 {
   "PSNProfiles": {
     "domain": "psnprofiles.com",
-    "contact": {
-      "facebook": "PSNProfiles",
-      "twitter": "PSNProfiles",
-      "email": "contact@psnprofiles.com"
-    },
+    "tfa": [
+      "totp"
+    ],
+    "notes": "2FA is configured via your linked PSNProfiles forum profile settings.",
```

Comment:
If you go to profile settings on the main PSNProfiles site, does it provide any indication of the security settings?

Reply:
The site is pretty light on details and no docs that I could find.  Clicking the player icon > settings button from the main site shows a pop up window for basic cosmetic profile changes, it has this banner at the top...

<img width="895" alt="Screenshot 2024-09-01 at 06 11 59" src="https://github.com/user-attachments/assets/7344c69d-2285-4f59-a9b9-1f513dd09f17">

Clicking that link takes you to forum settings where you can change your password, enable 2FA etc for the main site. It's not necessary to configure a forum profile to change security settings, which are shared between the forum and main site.

<img width="1282" alt="Screenshot 2024-09-01 at 06 13 14" src="https://github.com/user-attachments/assets/298cf142-ae98-45af-92a4-8dfcdb51697e">